src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,336 @@
+---
+part_number: II
+chapter_number: 13
+title: Capabilities
+description: Capabilities chapter of the 2020 Web Almanac covering brand-new, powerful web platform APIs.
+authors: [christianliebel]
+reviewers: [tomayac]
+analysts: [tomayac]
+translators: []
+discuss: 2049
+results: https://docs.google.com/spreadsheets/d/1N6j1qKv7vc51p1W9z_RrbJX9Se3Pmb-Uersfz4gWqqs/
+queries: 13_Capabilities
+christianliebel_bio: Christian Liebel is a consultant at <a href="https://thinktecture.com">Thinktecture</a>, supporting clients from various business areas in implementing first-class web applications. He is a Microsoft MVP for Developer Technologies, Google GDE for Web/Capabilities and Angular, and participates in the W3C Web Applications Working Group.
+featured_quote: The state of web capabilities in 2020 is healthy, as new powerful APIs regularly ship with new releases of Chromium-based browsers. The first APIs have already made their way into other browsers as well.
+featured_stat_1: 0.0006%
+featured_stat_label_1: (Chrome) Page loads using Async Clipboard API
+featured_stat_2: 0.49%
+featured_stat_label_2: Sites using Storage Manager API
+featured_stat_3: 363
+featured_stat_label_3: Sites allow installing related apps
+published: 2020-11-09T00:00:00.000Z
+last_updated: 2020-11-09T00:00:00.000Z
+unedited: true
+---
+
+## Introduction
+
+[Progressive Web Apps](./pwa) (PWA) are a cross-platform application model based on web technology. With the help of Service Workers, these applications run even when the user is offline. The Web App Manifest allows users to add a PWA to their home screen or program list. When opened from there, a PWA appears as a native application. However, PWAs can only use the functions and capabilities that are exposed through web platform APIs. Arbitrary native interfaces cannot be called, leaving a gap between native applications and web apps.
+
+The [Capabilities Project](https://www.chromium.org/teams/web-capabilities-fugu), informally also known as Project Fugu, is a cross-company effort by Google, Microsoft, and Intel to bridge the gap between web and native. To do so, the Chromium contributors implement new APIs exposing capabilities of the operating system to the web, while maintaining user security, privacy, and trust. These capabilities include, but are not limited to:
```

Comment:
Can you talk about why it's important to bridge the gap between web and native? I think the answer has profound implications on the future of the web. Maybe it's too big a question to answer here, but it's something I generally don't see discussed enough. This chapter and Mobile Web are probably the best places to have that discussion, even briefly.

Reply:
Added a brief sentence. Not sure if this suffices. 😇